tests/security/atsec/syscall_thrasher.pm

```diff
@@ -18,28 +18,24 @@ sub run {
 
     select_console 'root-console';
 
-    # Complile thrash.c
     my $exe_file = 'thrash';
     assert_script_run('cd /usr/local/atsec');
     assert_script_run("gcc -o $exe_file thrash.c");
 
-    # Change the permission of test script because we need to run it as non-root
     assert_script_run("chmod 755 $exe_file");
 
     # The test needs to run by non-root
     select_console 'user-console';
 
-    # Prepare the test directory
+    my $log_file = '/tmp/syscalls_output.log';
     my $test_dir = 'test_syscall_thrasher';
     assert_script_run("mkdir -p $test_dir");
 
-    # Copy the executable file to test directory
     assert_script_run("cp /usr/local/atsec/$exe_file $test_dir/");
 
-    # Run the test
     assert_script_run("cd $test_dir");
-    assert_script_run("./$exe_file", timeout => 1200);
-    assert_script_run('ls');
+    assert_script_run("./$exe_file >> $log_file", timeout => 900);
+    upload_logs("$log_file");
```

Comment:
This means that on failure it will not upload logs

Reply:
good point! let me fix that